sdk/textanalytics/ai-text-analytics/test/public/textAnalyticsClient.spec.ts

```diff
@@ -764,16 +764,16 @@ matrix([["AAD", "APIKey"]] as const, async (authMethod: AuthMethod) => {
             );
           });
 
-          // it("family emoji wit skin tone modifier", async function(this: Context) {
-          //   await checkOffsetAndLength(
-          //     client,
-          //     "👩🏻‍👩🏽‍👧🏾‍👦🏿 SSN: 859-98-0987",
-          //     "Utf16CodeUnit",
-          //     25,
-          //     11,
-          //     checkEntityTextOffset
-          //   );
-          // });
+          it("family emoji wit skin tone modifier", async function(this: Context) {
```

Comment:
Nit: the code duplication here could be reduced using `matrix` or just wrapping the tests and their `describe` blocks in a `for (const encoding of ["TextElement_v8", "Utf16CodeUnit", "UnicodeCodePoint"] as const) { ... }`

Reply:
Yeah but the expected values are different for each one. We could probably rely on `matrix` to set the expected values but I am afraid that could be unreadable.